src/components/custom/_dnumbers.js

```diff
@@ -10,6 +10,10 @@ const NumberWrapper = styled.div`
     flex-direction: column;
     justify-content: center;
     width: 32.8rem;
+    @media ${device.tabletL} {
+        padding: 1.2rem;
```

Comment:
Due to the global font-size configuration of the deriv-com project that changes over screen width sizes, as can be found in [`global-style.js`](https://github.com/binary-com/deriv-com/blob/master/src/themes/global-style.js), setting `rem` in anything below laptop width will not result in 1rem = 10px. Hence, we usually use `px` for anything equal and below laptop screen widths.